src/haddock/modules/analysis/caprieval/capri.py

```diff
@@ -34,12 +34,50 @@
     )
 from haddock.libs.libio import write_dic_to_file, write_nested_dic_to_file
 from haddock.libs.libontology import PDBFile, PDBPath
+from haddock.modules import get_module_steps_folders
+
+CNS_MODULES = ["rigidbody",
```

Comment:
what about putting this `CNS_MODULES` list in `src/haddock/core/defaults.py` ?

Reply:
good point, done it!